charts/container-storage-modules/Chart.yaml

```diff
@@ -30,25 +30,45 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 1.0.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.1.0"
+appVersion: "1.0.0"
 
 dependencies:
 - name: csi-powerstore
   version: 2.6.0
   repository: file://../csi-powerstore
   condition: csi-powerstore.enabled
 
+- name: csi-powermax
+  version: 2.6.0
+  repository: file://../csi-powermax
```

Comment:
This is fine for testing, but when the chart is released it will need to contain the full path to our helm repository. See example: https://github.com/dell/helm-charts/pull/195